src/Webauthn.h

```diff
@@ -0,0 +1,15 @@
+// Copyright © 2017-2023 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Data.h"
+
+namespace TW::Webauthn {
+
+Data getPublicKey(const Data& attestationObject);
```

Comment:
do we only support public key on nistp256? we might return `PublicKey` class instead